packages/gafl-webapp-service/src/handlers/__tests__/analytics-handler.spec.js

```diff
@@ -17,35 +17,41 @@ describe('The analytics handler', () => {
     jest.clearAllMocks()
   })
 
-  it('redirects to /buy', async () => {
+  it.each([
+    ['https://localhost:3000', 'https://localhost:3000/buy/name', '/buy/name'],
+    ['https://localhost:1234', 'https://localhost:1234/example/test/redirect', '/example/test/redirect'],
+    ['https://testserver-example-fish', 'https://testserver-example-fish/buy/renew/identify', '/buy/renew/identify']
+  ])('redirects to correct page', async (origin, referer, redirect) => {
     const payload = {
       analyticsResponse: 'accept'
     }
-    const request = generateRequestMock(payload)
+    const request = generateRequestMock(payload, 'analytics', origin, referer)
```

Comment:
I notice in these tests we sometimes have `payload` and `analytics` in single quotes and sometimes not – is there a reason we alternate?